src/mon/OSDMonitor.cc

```diff
@@ -2421,6 +2421,9 @@ bool OSDMonitor::prepare_boot(MonOpRequestRef op)
       }
     }
 
+    // update osd beacon timestamp
```

Comment:
@hjwsm1989 I believe It would be even better if we start comment with capital letter something as:
`//Update osd beacon timestamp`